src/NUnitConsole/nunit3-console.tests/ResultReporterTests.cs

```diff
@@ -101,8 +101,9 @@ public void SummaryReportTest()
             var expected = new string[] {
                 "Test Run Summary",
                 "    Overall result: Failed",
-                "   Tests run: 35, Passed: 30, Errors: 1, Failures: 3, Inconclusive: 1",
-                "     Not run: 10, Invalid: 3, Ignored: 4, Explicit: 3, Skipped: 0",
+                "   Tests run: 35, Passed: 30, Failed: 7, Inconclusive: 1, Skipped: 7",
+                "     Failed Tests - Failures: 3, Errors: 1, Invalid: 3",
+                "     Skipped Tests - Ignored: 4, Explicit: 3, Other: 0",
                 "  Start time: 2015-10-19 02:12:28Z",
                 "    End time: 2015-10-19 02:12:29Z",
                 "    Duration: 0.349 seconds",
```

Comment:
It looks like we had it a bit messed up before, but it is more obvious now with the new format. I would like to see the indenting fixed here and in nunitlite. I would like Overall, Tests run and Start time indented 2 spaces and the nodes under them indented 4 spaces.